addons/website_event/views/event_event_views.xml

```diff
@@ -74,6 +79,16 @@
                     </field>
                 </page>
             </page>
+              
+            <field name="tag_ids" position="attributes">
+                <attribute name="domain">['|', ('event_tag_website_id', '=', website_id), ('event_tag_website_id', '=', False)]</attribute>
+                <attribute name="attrs">{'invisible': [('website_id', '=', False)]}</attribute>
+            </field>
+
+            <field name="tag_ids" position="after">
+                <field name="tag_ids" widget="many2many_tags" options="{'color_field': 'color', 'no_quick_create': True}" attrs="{'invisible': [('website_id', '!=', False)]}"></field>
```

Comment:
Why would we have two tag_ids fields defined ?

Reply:
@tde-banana-odoo because when website is set to Null (False) i have to show all the tags (i don't have to filter them). So the second field will only work if website ID is False (according to attributes)
![image](https://github.com/odoo/odoo/assets/52849166/4945c7ef-bd28-4e52-b8d9-8ca4af4bf7eb)